libs/pages/application/src/lib/ui/page-settings-advanced/page-settings-advanced.tsx

```diff
@@ -0,0 +1,186 @@
+import { ApplicationAdvancedSettings } from 'qovery-typescript-axios'
+import { useState } from 'react'
+import { Controller, useFormContext } from 'react-hook-form'
+import { LoadingStatus } from '@console/shared/interfaces'
+import {
+  CopyToClipboard,
+  EditionTable,
+  EditionTableRow,
```

Comment:
Isn't it better if we prefix the components name with the important element and then put its differentiator?

Like: `TableEdition` and `TableRowEdition` like this we see directly it's a table component?